gsl_projet/templates/includes/projet_detail/_dotation_projet_financial_informations.html

```diff
@@ -0,0 +1,16 @@
+{% load gsl_filters %}
+<li>
+    <span aria-hidden="true">💰</span> Assiette subventionnable : {{ dotation_projet.assiette|euro }}
+</li>
+<li>
+    <span aria-hidden="true">🧾</span> Taux de subvention sollicité : {{ dotation_projet.taux_de_subvention_sollicite|percent|default:"—" }}
+    <button class="fr-btn--tooltip fr-btn fr-m-0 fr-pv-0-5v gsl-tooltip"
+            type="button"
+            aria-describedby="tooltip-taux-{{ dotation_projet.dotation }}">
+        en % de l'assiette subventionnable
```

Comment:
J'ai testé qq trucs, je pense que le plus utile pour les utilisateurs de lecteurs d'écran serait ceci : 
```suggestion
        information contextuelle : en % de l'assiette subventionnable
```

Le DSFR proposait de mettre uniquement "information contextuelle" et dans ce cas on avait l'info "en % de l'assiette subventionnable" seulement en cliquant sur le bouton.

Si on met juste "en % de l'assiette subventionnable" ici, ça donne un truc un peu bizarre où on ne comprend pas à quoi sert le bouton, je trouve.